virtualization/windowscontainers/docker/configure_docker_daemon.md

```diff
@@ -110,6 +110,14 @@ Only the desired configuration changes need to be added to the configuration fil
 }
 ```
 
+Likewise below example can configure docker engine to keep images and containers to different path other than c:\programdata\docker.
```

Comment:
For consistency:

`Likewise, this sample configures the Docker daemon to keep images and containers in an alternate path. If not specified, the default is c:\programdata\docker.`